specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/secrets.json

```diff
@@ -116,6 +116,94 @@
             "$ref": "./examples/DeleteSecret-example.json"
           }
         }
+      },
+      "get": {
```

Comment:
I think this is going to cause problems. The REST API already treats the `secret-version` as optional in the `/secrets/{secret-name}/{secret-version}` endpoint, and the SDK reflects this. If you add an operation like this, it will generate an overload that won't compile. Why this change? It seems unnecessary even from a pure REST client standpoint since `secret-version` is optional already.

Reply:
oh. This explains why there is no get operation under path /secrets/{secret-name}. I have removed it now and updated it in the getting secret version operation